test_environments/message_and_event_brokers/kafka/no_authentication/docker-compose.yml

```diff
@@ -0,0 +1,25 @@
+---
+version: '3'
+services:
+  zookeeper:
+    image: confluentinc/cp-zookeeper:7.3.2
+    container_name: zookeeper
+    environment:
+      ZOOKEEPER_CLIENT_PORT: 2181
+      ZOOKEEPER_TICK_TIME: 2000
+
+  broker:
+    image: confluentinc/cp-kafka:7.3.2
+    container_name: broker
+    ports:
+      - "9092:9092"
+    depends_on:
+      - zookeeper
```

Comment:
This can be easily added for every setup except SASL_SCRAM.
SASL_SCRAM requires that user for Kafka Broker is added to Zookeeper  before Kafka broker is started.
I will have to investigate if this can be automated somehow. 
Maybe we can work on that additionally? WDYT @m-vdb @ancalita

Reply:
Note: Added health check for every setup except SASL_SCRAM.

Reply:
I think a note explaining this in relevant README is more than enough for a first go